prisma/schema.prisma

```diff
@@ -12,57 +12,185 @@ datasource db {
     // Further reading:
     // https://next-auth.js.org/adapters/prisma#create-the-prisma-schema
     // https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#string
-    url      = env("DATABASE_URL")
+    url       = env("DATABASE_URL")
     directUrl = env("DIRECT_URL")
 }
 
-model Example {
-    id        String   @id @default(cuid())
-    createdAt DateTime @default(now())
-    updatedAt DateTime @updatedAt
+model Reviews {
+    id                   Int            @id @default(autoincrement())
+    body                 String
+    tips                 String?
+    rating               Int
+    reviewedProfessor    Professors     @relation(fields: [reviewedProfessorId], references: [id])
+    reviewedProfessorId  Int            @map("reviewed_professor")
+    reviewedCourse       Courses        @relation(fields: [reviewedCourseId], references: [id])
+    reviewedCourseId     Int            @map("reviewed_course")
+    reviewerUniveristy   Universities   @relation(fields: [reviewerUniveristyId], references: [id])
+    reviewerUniveristyId Int            @map("reviewer_university")
+    reviewerFaculty      Faculties      @relation(fields: [reviewerFacultyId], references: [id])
+    reviewerFacultyId    Int            @map("reviewer_faculty")
+    reviewer             Users          @relation(fields: [reviewerId], references: [id])
+    reviewerId           Int            @map("reviewer")
+    createdAt            DateTime       @default(now()) @map("created_at") @db.Timestamptz(3)
+    updatedAt            DateTime       @default(now()) @updatedAt @map("updated_at") @db.Timestamptz(3)
+    Votes                Votes[]
+    ReviewLabels         ReviewLabels[]
 }
 
-// Necessary for Next auth
-model Account {
-    id                String  @id @default(cuid())
-    userId            String
-    type              String
-    provider          String
-    providerAccountId String
-    refresh_token     String? @db.Text
-    access_token      String? @db.Text
-    expires_at        Int?
-    token_type        String?
-    scope             String?
-    id_token          String? @db.Text
-    session_state     String?
-    user              User    @relation(fields: [userId], references: [id], onDelete: Cascade)
+model Users {
+    id           Int          @id @default(autoincrement())
+    username     String       @unique
+    firstName    String       @map("first_name")
+    lastName     String       @map("last_name")
```

Comment:
I think username should be enforced so that we can identify each account by a unique name. Or do you have anything in mind other than a username? (eg. random acc ID? etc)

Agree that first name and last name can be optional, and this is addressed in the latest commit

Reply:
conclusion after discussion with the team:
- we should enforce `username` (maybe randomly generate one and let the user change values)
- `first_name`, `last_name` should be optional